client/securedrop_client/gui/conversation/export/export_wizard_page.py

```diff
@@ -0,0 +1,498 @@
+        """
+        Provide setter for auto-advancing wizard past the Preflight page.
+        If True, as soon as a Status is available, the wizard will advance
+        to the appropriate page.
+        """
+        self._should_autoskip_preflight = should_autoskip
+
+    def should_autoskip_preflight(self) -> bool:
+        """
+        Return True if Preflight page should be advanced automatically as soon as
+        a given status is available.
+
+        This workaround exists to let users skip past the preflight page if they are
+        returned to it from a later page. This is required because in PyQt5,
+        QWizard cannot navigate to a specific page, meaning users who insert an
+        unlocked drive, then start the wizard, then encounter a problem are sent
+        "back" to this page rather than to the InsertUSBPage, since it wasn't in
+        their call stack.
+
+        The autoskip combined with custom nextId logic in ExporWizardPage allows us
+        to emulate the desired behaviour.
+        """
+        return self._should_autoskip_preflight
+
+    @pyqtSlot(object)
+    def on_status_received(self, status: ExportStatus) -> None:
+        self.status = status
+        self.stop_animate_header()
+        header = _("Ready to export:<br />" '<span style="font-weight:normal">{}</span>').format(
+            self.summary
+        )
+        self.header.setText(header)
+        self.completeChanged.emit()
+
+        if self.wizard() and isinstance(self.wizard().currentPage(), PreflightPage):
+            # Let users skip preflight screen if they have already seen it. The first time a status
+            # is received, autoskip is False, and a user has to manually click "Continue";
+            # after that, it's True.
+            if self.should_autoskip_preflight():
+                self.wizard().next()
+            else:
+                self.set_should_autoskip_preflight(True)
+
+
+class ErrorPage(ExportWizardPage):
+    def __init__(self, export: Export):
+        header = _("Export Failed")
+        super().__init__(export, header=header, body=None)
+
+    def isComplete(self) -> bool:
+        """
+        Override isComplete() to always return False. This disables
+        the 'next' button on the error page and means users can
+        only go back to a previous page or exit the wizard.
+        """
+        return False
+
+    @pyqtSlot(object)
+    def on_status_received(self, status: ExportStatus) -> None:
+        self.status = status
+
+
+class InsertUSBPage(ExportWizardPage):
+    def __init__(self, export: Export, summary: str) -> None:
+        self.no_device_hint = 0
+        self.summary = summary
+        header = _("Ready to export:<br />" '<span style="font-weight:normal">{}</span>').format(
+            summary
+        )
+        body = _(
+            "Please insert one of the export drives provisioned specifically "
+            "for the SecureDrop Workstation."
```

Comment:
Oh no - so sorry I hadn't explained that better.  I have a hint message in the unlock error (that you may not have caught since I pushed it recently): https://github.com/freedomofpress/securedrop-client/pull/1777/files#diff-60a2f9ad004323b42fa3c5304f210bd632ef6c0d0a5be5b923abb1a36bbbb871R29-R31 

If you think we should put it front and centre though I can also make that change.

Reply:
I lean towards proactive documentation (or just cuing, in this case) for fewer surprises.  But I defer to you!

Reply:
(Updated in https://github.com/freedomofpress/securedrop-client/commit/05555203b8a768dfab9ab584f424c1838c0b6ad4)